__test__/app.test.js

```diff
@@ -72,22 +119,6 @@ describe('/api', () => {
         });
     });
 
-    describe('GET all tables', () => {
-      it('SUCCESS - status 200 - returns all tables', () => {
-        return request(app)
-          .get('/api/tables')
-          .expect(200)
-          .then(({ body }) => {
-            expect(body.tables).toHaveLength(8);
-            body.tables.forEach(table => {
-              expect(table).toEqual(
-                expect.objectContaining({
-                  table_id: expect.any(Number),
-                  name: expect.any(String),
```

Comment:
I  can see this has been replaced with only the is_active in the below objectContaining statement. This shouldn't have been edited.